From a1ba3de0bfc9f73acb9c666129b20527a6ce67a1 Mon Sep 17 00:00:00 2001
From: KWottrich <kenny.wottrich@gmail.com>
Date: Wed, 11 Oct 2023 10:37:24 -0500
Subject: [PATCH] temp change to PHYS for testing

---
 src/joystick/linux/SDL_sysjoystick.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/joystick/linux/SDL_sysjoystick.c b/src/joystick/linux/SDL_sysjoystick.c
index 66ff60cace18..82f0d3c9a823 100644
--- a/src/joystick/linux/SDL_sysjoystick.c
+++ b/src/joystick/linux/SDL_sysjoystick.c
@@ -1409,7 +1409,7 @@ static SDL_sensorlist_item *GetSensor(SDL_joylist_item *item)
     if (fd_item < 0) {
         return NULL;
     }
-    if (ioctl(fd_item, EVIOCGUNIQ(sizeof(uniq_item) - 1), &uniq_item) < 0) {
+    if (ioctl(fd_item, EVIOCGPHYS(sizeof(uniq_item) - 1), &uniq_item) < 0) {
         return NULL;
     }
     close(fd_item);
@@ -1430,7 +1430,7 @@ static SDL_sensorlist_item *GetSensor(SDL_joylist_item *item)
         if (fd_sensor < 0) {
             continue;
         }
-        if (ioctl(fd_sensor, EVIOCGUNIQ(sizeof(uniq_sensor) - 1), &uniq_sensor) < 0) {
+        if (ioctl(fd_sensor, EVIOCGPHYS(sizeof(uniq_sensor) - 1), &uniq_sensor) < 0) {
             close(fd_sensor);
             continue;
         }
